src/node-red/index.njk

```diff
@@ -162,12 +129,22 @@ sitemapPriority: 0.8
             </div>
         </div>
     </div>
+</div>    
 
-    <div class="text-white">
-        {% include "components/dividers/light-grey-curve.svg" %}
-    </div>
+<!--Customer Stories Banner-->
+<a href="/customer-stories/" class="w-full bg-teal-700 text-center text-teal-50 px-6 py-3 text-lg flex flex-row items-center justify-center gap-1 hover:bg-teal-800 hover:no-underline cursor-pointer flex-wrap">
+    Check out some
+    <span class="font-bold hover:no-underline"> Node-RED </span> 
```

Comment:
Having the spaces inside the span I don't think is needed?

Reply:
You're right, the gap-1 does the work, I removed the spaces